### PR TITLE
feat: documentation rework 

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
The EICrecon repo should not serve as the source of documentation for unrelated components. It is not updated when those other components are updated, and therefore likely wrong in several cases.

List of changes:
- Remove `edpm` since not a supported or sustainable project.
- Removed unsupported direct singularity calls to cvmfs since they tend to fail without additional flags.
- Removed non-standard singularity installation instructions that pull a `.deb` file from the internals of an apt dist server.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [x] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [x] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No.

### Does this PR change default behavior?
No.

### CODE REVIEW

**Comment by DraTeots** [comment]
When software group decides on the documentation website, the unrelated components will go from eicrecon documentation. For now it is the only place where common questions of EICrecon users can be addressed

**Review by DraTeots — CHANGES_REQUESTED** [review]

File: docs/howto/use_singularity.md [hunk 1]
```diff
@@ -31,10 +18,10 @@ laptops). It also creates eic\_shell with the right environment setup,
 prepares the current dir to work with detector or etc.
 
 ```bash
-curl https://eicweb.phy.anl.gov/containers/eic_container/-/raw/master/install.sh | bash
+curl -L https://get.epic-eic.org | bash
```
[the remark this answers is not in the document]
Comment (wdconinc) [reply docs/howto/use_singularity.md:21]: It works now. It was fixed based on the issue I reported.

File: docs/howto/use_singularity.md [hunk 2]
```diff
@@ -6,20 +6,7 @@
...
-
-```bash
-sudo apt install containernetworking-plugins
-wget http://ftp.fi.debian.org/debian/pool/main/s/singularity-container/singularity-container_3.10.3+ds1-1_amd64.deb
-sudo dpkg -i singularity-container_3.10.3+ds1-1_amd64.deb
-```
-
-Please don\'t install NeuroDebian repo (first in google search) as its singularity v2.6
-and eic image require singularity \> 3.0.
+The [oficial installation instructions](https://docs.sylabs.io/guides/latest/user-guide/quick_start.html#quick-installation-steps) have many steps.
```
[the remark this answers is not in the document]
Comment (DraTeots) [reply docs/howto/use_singularity.md:9]: To clarify. I agree it is better to remove previous instructions to direct image download which updates over time. But the link above stays correct for at least 4 years already. So leave it like "If you have an ubuntu here you can find prebuilt deb packages"

[the remark this answers is not in the document]
Comment (wdconinc) [reply docs/howto/use_singularity.md:9]: We cannot point to a debian pool link and tell everyone to install from there, even if they use ubuntu "whatever compatible". Even if the link is stable, that's not the problem. The problem is that you are telling someone to install a package that was not built for their system.

```suggestion
If you use Debian sid, you may be able to install `singularity-container`. If you use Ubuntu or Red Hat, the `deb` and `rpm` packages at https://github.com/sylabs/singularity/releases/ may be useful.

The [official installation instructions](https://docs.sylabs.io/guides/latest/user-guide/quick_start.html#quick-installation-steps) may be useful for other operating systems.
```

**Review by DraTeots — APPROVED** [review]

**Pull request merged**